forem/forem
# Fix broken Podcast liquid tag

## What type of PR is this? (check all applicable)
- [x] Bug Fix for Forem.

## Description
the asset images in `app/views/podcast_episodes/_liquid.html.erb` were using the asset path without fingerprint, causing the tag to be broken Forem. This PR

1. Remove the redundant podcast bar component in `app/views/podcast_episodes/_liquid.html.erb` and instead, call `render the partial "podcast_episodes/podcast_bar"` directly.
2. Replace all uses of instant variables in podcast bar partial with locals variable for maintainability and reusability.

## Related Tickets & Documents
Resolves https://github.com/forem/forem/issues/11462

## QA Instructions, Screenshots, Recordings
0. Pull down this PR, make sure you have podcasts locally with background job running and caching enabled.
1. Start up the rails server
2. head to `/pod` and choose a podcast episode.
3. head to `/new` and use the liquid tag in this manner in the body, `{% podcast localhost:3000/your/podcast/link %}`
4. Click save. You should see all icons like play/pause rendered normally. When playing the podcast you should see the volume and mute button working normally.

## Added tests?
- [x] No, and this is why: existing test covers this.

## Added to documentation?

## [optional] Are there any post deployment tasks we need to perform?
n/a
